src/shared/cryptsetup-util.c

```diff
@@ -304,6 +307,9 @@ int dlopen_cryptsetup(void) {
 #if HAVE_CRYPT_REENCRYPT_INIT_BY_PASSPHRASE
                         DLSYM_ARG(crypt_reencrypt_init_by_passphrase),
 #endif
+#if HAVE_CRYPT_REENCRYPT_RUN
+                        DLSYM_ARG(crypt_reencrypt_run),
+#endif
 #if HAVE_CRYPT_REENCRYPT
```

Comment:
Maybe, better to load the old one only when the new one is not detected? Then, we can drop the suppression of deprecated warning, though I have not tested.